docs/resources.rst

```diff
@@ -106,7 +106,7 @@ Solidity Tools
     Tool to generate Solidity interface source from a given ABI JSON.
 
 * `Aderyn <https://github.com/Cyfrin/aderyn>`_
-    Rust-based solidity smart contract static analyzer designed to help find vulnerabilities in Solidity code bases.
+    Rust-based solidity smart contract static analyzer designed to help find vulnerabilities in Solidity code bases. Comes with `VS Code Extension <https://marketplace.visualstudio.com/items?itemName=Cyfrin.aderyn>` for easy use.
```

Comment:
Can you move this up to the `Visual Studio Code (VS Code)` section? Insert it at the top to maintain alphabetical ordering.